.github/sync.yml

```diff
@@ -9,7 +9,11 @@ group:
     jhudsl/Documentation_and_Usability_Quizzes
     broadinstitute/dsp-terra-mooc-intro_Quizzes
     jhudsl/Computing_for_Cancer_Informatics_Leanpub
-
+    fhdsl/Choosing_Genomics_Tools_Quizzes
+    fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes
+    fhdsl/Overleaf_and_LaTeX_for_Scientific_Articles_Quizzes
+    fhdsl/AI_for_software_Quizzes
+    jhudsl/Informatics_Research_Leadership_Leanpub
```

Comment:
If the Leadership GitHub repo is up to date then it wouldn't transfer updated material unless this repo is called _Quizzes instead of _Leanpub

Reply:
OK I will change the names of the _Leanpub repos

Reply:
That one is especially out- of-date cause it predated so much infrastructure - I don't think it has the iframes

Reply:
tech debt!